tests/conftest.py

```diff
@@ -100,3 +100,19 @@ def dummy_config(test_seed, dummy_split, dummy_dataset_name, dummy_model_name):
         "n_samples": 50,
         "random_state": test_seed,
     }
+
+
+@pytest.fixture
+def dummy_validation_scores(rng, test_n_samples):
+    "Sort here to induce uncorrelation with dummy_random_metric_scores"
```

Comment:
```suggestion
    # Sort here to induce uncorrelation with dummy_random_metric_scores
```

Prefer this as a comment rather than a hanging string but will commit directly